docs/nidcpower/attributes.rst

```diff
@@ -2187,6 +2377,138 @@ nidcpower.Session properties
             - LabVIEW Property: **Source:Pulse Voltage:Pulse Bias Current Limit**
             - C Attribute: **NIDCPOWER_ATTR_PULSE_BIAS_CURRENT_LIMIT**
 
+.. py:attribute:: pulse_bias_current_limit_high
```

Comment:
These correspond to a new NI-DCPower (17.6.1 I believe). We should at least not it in the CHANGELOG.md and the PR's description.